### PR TITLE
Audit fixes

### PR DESCRIPTION
**Pull request by GWSzeto**
Addresses the changes found in the audit prelim report for modular contracts

### CODE REVIEW

File: src/module/token/tokenId/SequentialTokenIdERC1155.sol [hunk 1]
```diff
@@ -73,6 +75,29 @@ contract SequentialTokenIdERC1155 is Module, UpdateTokenIdCallbackERC1155 {
...
+
+    /// @dev Called by a Core into an Module during the uninstallation of the Module.
+    function onUninstall(bytes calldata data) external {}
+
+    /*//////////////////////////////////////////////////////////////
+                    Encode install / uninstall data
+    //////////////////////////////////////////////////////////////*/
+
+    /// @dev Returns bytes encoded install params, to be sent to `onInstall` function
+    function encodeBytesOnInstall(uint256 nextTokenId) external pure returns (bytes memory) {
```
Comment (joaquim-verges) [inline src/module/token/tokenId/SequentialTokenIdERC1155.sol:92]: Would call this startTokenId to be clear of what it is

File: src/module/token/metadata/BatchMetadataERC721.sol [hunk 2]
```diff
@@ -14,12 +14,10 @@ library BatchMetadataStorage {
         keccak256(abi.encode(uint256(keccak256("token.metadata.batch")) - 1)) & ~bytes32(uint256(0xff));
 
     struct Data {
-        // tokenId range end
-        uint256[] tokenIdRangeEnd;
         // next tokenId as range start
         uint256 nextTokenIdRangeStart;
-        // tokenId range end => baseURI of range
-        mapping(uint256 => string) baseURIOfTokenIdRange;
+        // tokenId range end
```
Comment (joaquim-verges) [inline src/module/token/metadata/BatchMetadataERC721.sol:19]: Update comment?

File: src/module/token/metadata/BatchMetadataERC721.sol [hunk 3]
```diff
@@ -130,71 +128,42 @@ contract BatchMetadataERC721 is Module, UpdateMetadataCallbackERC721 {
...
         revert BatchMetadataNoMetadataForTokenId();
     }
 
-    /// @dev Sets the base URI for the batch of tokens with the given batchId.
-    function setBaseURI(uint256 _batchId, string memory _baseURI) external virtual {
-        _batchMetadataStorage().baseURIOfTokenIdRange[_batchId] = _baseURI;
-        (uint256 startTokenId,) = getBatchRange(_batchId);
-        emit BatchMetadataUpdate(startTokenId, _batchId);
+    /// @dev Sets the base URI for the batch based on the batchIndex.
+    function setBaseURI(uint256 _batchIndex, string memory _baseURI) external virtual {
```
Comment (joaquim-verges) [inline src/module/token/metadata/BatchMetadataERC721.sol:162]: Would help me to have a list of ABI changes, hard to see from this pr

File: src/module/token/metadata/BatchMetadataERC721.sol (hunk 2 above)
Comment (GWSzeto) [reply src/module/token/metadata/BatchMetadataERC721.sol]: addressed

File: src/module/token/tokenId/SequentialTokenIdERC1155.sol (hunk 1 above)
Comment (GWSzeto) [reply src/module/token/tokenId/SequentialTokenIdERC1155.sol]: addressed

**Review by kumaryash90 — APPROVED** [review]

**Pull request merged**